build.js

```diff
@@ -14,12 +13,23 @@ const ajv = new Ajv({
   allErrors: true,
 });
 
+// need to define our custom validation name variable
+ajv.addKeyword({
+  keyword:'$registry-validationFunctionName',
+  schemaType: 'string'
+});
```

Comment:
let's worry about changing these in a separate PR - stick with the existing import format